app_dart/test/request_handlers/update_discord_status_test.dart

```diff
@@ -0,0 +1,252 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+
+import 'package:cocoon_server_test/test_logging.dart';
+import 'package:cocoon_service/src/model/firestore/task.dart';
+import 'package:cocoon_service/src/request_handlers/update_discord_status.dart';
+import 'package:cocoon_service/src/service/build_status_provider.dart';
+import 'package:cocoon_service/src/service/discord_service.dart';
+import 'package:googleapis/firestore/v1.dart';
+import 'package:mockito/mockito.dart';
+import 'package:test/test.dart';
+
+import '../src/datastore/fake_config.dart';
+import '../src/request_handling/request_handler_tester.dart';
+import '../src/utilities/entity_generators.dart';
+import '../src/utilities/mocks.mocks.dart';
+
+void main() {
+  useTestLoggerPerTest();
+
+  late RequestHandlerTester tester;
+  late UpdateDiscordStatus handler;
+  late MockFirestoreService mockFirestoreService;
```

Comment:
Can you use `FakeFirestoreService` instead of `MockFirestoreService`? I'm trying to actively get rid of the latter, which does not need to be used anymore because similar to Datastore we have a full in-memory Fake (with real filter validation, etc).

Reply:
I should have said "in a future pr" - but here I am..